.changeset/wicked-sheep-drive.md

```diff
@@ -0,0 +1,5 @@
+---
+"@optiaxiom/react": patch
+---
+
+Allow consumers to trigger `Autocomplete` with toggle button
```

Comment:
let's re-word this a bit - maybe say "added toggle button to autocomplete trigger" or something